recipe/test-rpath-nvcc.sh

```diff
@@ -2,7 +2,14 @@
 
 [[ ${target_platform} == "linux-64" ]] && targetsDir="targets/x86_64-linux"
 [[ ${target_platform} == "linux-ppc64le" ]] && targetsDir="targets/ppc64le-linux"
-[[ ${target_platform} == "linux-aarch64" ]] && targetsDir="targets/sbsa-linux"
+# https://docs.nvidia.com/cuda/cuda-compiler-driver-nvcc/index.html?highlight=tegra#cross-compilation
+[[ ${target_platform} == "linux-aarch64" && ${arm_variant_type} == "sbsa" ]] && targetsDir="targets/sbsa-linux"
+[[ ${target_platform} == "linux-aarch64" && ${arm_variant_type} == "tegra" ]] && targetsDir="targets/aarch64-linux"
```

Comment:
Just a note, while `arm_variant_type` is set in the context of this build, it is defined. However a downstream build that uses `cuda-nvcc_{{ target_platform }}` (or `{{ compiler("cuda") }}` in `requirements/build` or a user installing this in a Conda environment, may not have `arm_variant_type` set. To fix this PR ( https://github.com/conda-forge/cuda-nvcc-feedstock/pull/57 ) adds `arm_variant_type` to the compiler activation script so that this is also set